admin/services/check-window-v2-update.service.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const checkWindowDataService = require('./data-access/check-window.data.service')
+const checkWindowAddValidator = require('../lib/validator/check-window-v2/check-window-add-validator')
+const checkWindowHelper = require('../helpers/check-window')
+const checkWindowV2Service = require('./check-window-v2.service')
+const monitor = require('../helpers/monitor')
+
+const checkWindowV2UpdateService = {}
+
+/**
+ * Submit request data on updating
+ * @param {Object} requestData
+ * @returns {String} flash message for successful db insertion
+ */
+checkWindowV2UpdateService.submit = async (requestData) => {
+  const checkWindow = await checkWindowV2Service.getCheckWindow(requestData.checkWindowUrlSlug)
+  const validationError = checkWindowAddValidator.validate(requestData)
+  if (validationError.hasError()) {
+    throw validationError
+  }
+  const checkWindowData = checkWindowHelper.prepareSubmissionData(requestData, checkWindow.id)
```

Comment:
This is odd. The Presentation helper is now being called by a service.  Which tier of the application is the Helper actually in?  Either Presentation or Business logic service.

Reply:
@jon-shipley the method is moved under check window v2 service.